Kickstarter-iOS/Features/Comments/Controllers/CommentRepliesViewController.swift

```diff
@@ -189,6 +197,27 @@ extension CommentRepliesViewController: CommentComposerViewDelegate {
   }
 }
 
+// MARK: - CommentCellDelegate
+
+extension CommentRepliesViewController: CommentCellDelegate {
+  func commentCellDidTapHeader(_ cell: CommentCell, author _: Comment.Author) {
+    guard AppEnvironment.current.currentUser != nil, featureBlockUsersEnabled() else { return }
+
+    let actionSheet = UIAlertController
+      .blockUserActionSheet(
+        blockUserHandler: { _ in self.blockUser() },
+        sourceView: cell,
+        isIPad: self.traitCollection.horizontalSizeClass == .regular
+      )
+
+    self.present(actionSheet, animated: true)
+  }
+
+  func commentCellDidTapReply(_: CommentCell, comment _: Comment) {}
+
+  func commentCellDidTapViewReplies(_: CommentCell, comment _: Comment) {}
```

Comment:
If implementing these everywhere is annoying, you could provide a default empty implementation as an extension next to the protocol. (I don't feel strongly about this, though.)